public/content/documentation/native/controls/checkbox.md

```diff
@@ -0,0 +1,295 @@
+
+   - Name: Name describes the purpose of the control and matches the visible label
+   - Role: Identifies itself as a checkbox in Android and a Button or checkbox in iOS
+   - Group: Visible label can be grouped with the checkbox in a single swipe
+   - State: Expresses its state (disabled/dimmed, checked, not checked, selected, unselected)
+
+4. Test device settings
+
+   - Text resize: Text label can resize up to 200% without losing information
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/checkbox](https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/checkbox)
+
+## Gherkin
+
+### #a11y - Native Accessibility Acceptance Criteria
+
+How to test a checkbox
+
+GIVEN THAT I am on a screen with a checkbox
+
+1. Scenario: Test keyboard actions
+
+   - WHEN I press the "TAB", "ARROW KEYS", or "CTRL+TAB" keys 
+      - THEN the focus should visibly move to the checkbox 
+   - WHEN I press the "SPACEBAR" key 
+      - THEN the checkbox should be activated on iOS and Android 
+   - WHEN I press the "ENTER" key 
+      - THEN the checkbox should be activated on Android
+
+2. Scenario: Test mobile screen reader gestures
+
+   - WHEN I swipe to navigate to the checkbox 
+      - THEN the focus should move to the checkbox 
+         - AND the checkbox's name, role, and state should be expressed 
+   - WHEN I double-tap the checkbox 
+      - THEN the checkbox should toggle between CHECKED and NOT CHECKED states 
+
+3. Test screen reader output on all devices
+
+   - WHEN a screen reader reads the checkbox 
+      - THEN its name should describe the purpose of the control and match the visible label 
+         - AND its role should be identified as a checkbox in Android and as a button or checkbox in iOS 
+         - AND its visible label should be grouped with the checkbox in a single swipe 
+         - AND its state (DISABLED/DIMMED, CHECKED, NOT CHECKED, SELECTED, UNSELECTED) should be expressed
+
+4. Scenario: Test device OS settings for text resize
+
+   - WHEN I adjust the device text resize setting to 200% 
+      - THEN the text label should resize up to 200% without losing information 
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/checkbox](https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/checkbox)
+
+## iOS Developer Notes
+- A check box lets the user choose between two opposite states, actions or values  
+- You should use a native app control when at all possible vs a custom element, as it will automatically and correctly announce the role without additional development effort
+- A check box should just toggle between checked and unchecked.  It should not automatically navigate the user to another field or screen when activated, as that may cause a change of context.  Revealing new information on the same screen as a result of activating a checkbox is usually ok, as it is not a change of context
```

Comment:
Feel free to ignore this if you want: can we remove "just" from the first sentence and reduce the double spaces down to one space after each period?